backends/ubpf/ubpfControl.cpp

```diff
@@ -68,7 +68,33 @@ namespace UBPF {
             return;
         }
 
-        ::error("%1%: Not supported", function->method);
+        if (!control->emitExterns)
```

Comment:
It would be nice if you could somehow factor this code out in ebpf and just call it here - if it does not require too much work. p4c-xdp includes all of p4c-ebpf, and does this frequently, to minimize code duplication.

Reply:
I've refactored the code. It minimizes this particular code duplication, but it could be also a basis for further refactoring to minimize code duplication between p4c-ubpf and p4c-ebpf.